api/v1_projects.inc

```diff
@@ -799,6 +799,21 @@ function api_v1_project_validatetext($method, $data, array $query_params)
     return $text_checker->analyse(receive_project_text_from_request_body('text'), $data[":projectid"]);
 }
 
+function api_v1_project_pickersets($method, $data, $query_params)
+{
+    $project = $data[":projectid"];
+    $verbose_pickersets = [];
+    $pickersets = $project->get_pickersets();
+    foreach ($pickersets as $pickerset) {
+        $verbose_pickersets[] = [
+            "name" => $pickerset->name,
+            "subsets" => $pickerset->get_verbose_subsets(),
+            "titles" => $pickerset->get_titles(),
+        ];
```

Comment:
Would the return value for this be more useful to the client if it looked like:
```
$verbose_pickersets[] = [
    "name" => $pickerset->name,
    "subsets" =
        [
            "title" => "string",
            "characters" => [...],
        ],
        [
            "title" => "string2",
            "characters" => [...],
        ],
];
```
Or something similar? That way the client doesn't have to map the titles back to the subsets?

Reply:
Yes, thta would be better